src/index.html

```diff
@@ -1,35 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
+  <script src="script.js"></script>
 </head>
+
 <body>
+  <main id="lb-main">
     <h1>Leaderboard</h1>
-    <div class="board">
-        <div class="rec-scores">
-            <div class="scores1">
-                <div class="top1">
-                    <h2>Recent Scores</h2>
-                    <button class="refreshed">Refresh</button>
-                </div>
-                <div class="bottom1">
-                    <ul class="scorelist">
-                        
-                    </ul>
 
-                </div>
-            </div>
-        </div>
-        <div class="addscores1">
-            <div class="adding">
-                <h2>Add your score</h2><br><br>
-                <input class="name" type="text" placeholder="name"><br><br>
-                <input class="score" type="text" placeholder="score"><br><br>
-                <button class="add">Submit</button>
-            </div>
+    <div class="structure">
+      <section class="scores-section">
+        <div class="recent-scores">
+          <h2>Recentscores</h2>
+          <button type="submit" id="Refresh1">Refresh</button>
+          <ul id="scores-list"></ul>
         </div>
+
+        
+          <ul id="Table1">
+            
+          </ul>
+      </section>
+
+      <section id="form-section">
+        <h2>Add Your Scores</h2>
+
+        <form id="form">
+          <input type="text" placeholder="Your name" id="name">
+          <input type="number" placeholder="Your score" id="score">
+          <span id="error-msg">Kindly Complete All Fields!</span>
+          <button type="submit" id="btn">Submit</button>
+        </form>
```

Comment:
- [ ] Your work so far is impressive, @PrinceBadru. However, I wanted to bring to your attention that, as the previous reviewer mentioned, I am still unable to add a new score to your project. I have attached a video reference below that shows an error message in the console each time the submit button is pressed. This error might be the reason why your project is not working as expected. I kindly suggest that you fix this issue so that scores can be added to your project, allowing you to meet all the project requirements.. :100: 
- [ ] Another issue that the previous reviewer mentioned and that I would like to draw your attention to is the `Refresh button` not functioning as expected. As shown in the attached video, clicking the button has no effect. According to the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md), when the `Refresh button` is clicked, it should hit the `GET API endpoint`, causing the scores to update to include the newest score and display the old scores as well :wink: . 

[Screencast from 04-22-2023 11:20:08 AM.webm](https://user-images.githubusercontent.com/102699958/233781273-6a25461d-5251-4901-9ae5-9777dcdfe84f.webm)